test/lockdown/lockdown-specs.js

```diff
@@ -31,10 +31,19 @@ describe('lockdown', function () {
     await lockdown.getValue({ Key: 'ProductName'}, -1).should.eventually.be.rejected;
   });
 
-  it('lockdown query type', async function () {
+  it('should get lockdown query type', async function () {
     ({server, socket} = await getServerWithFixtures(fixtures.LOCKDOWN_QUERY_TYPE));
     const lockdown = new Lockdown(new PlistService(socket));
 
     await lockdown.queryType();
   });
+
+  it('should get device time', async function () {
+    ({server, socket} = await getServerWithFixtures(fixtures.LOCKDOWN_GET_VALUE_TIME));
+    const lockdown = new Lockdown(new PlistService(socket));
+    const epochValue = await lockdown.getValue({ Key: 'TimeIntervalSince1970' });
+    let date = new Date(0); // The 0 there is the key, which sets the date to the epoch
```

Comment:
This can be a `const` too. `let` is only needed if you're re-assigning a variable.

Reply:
Thanks for catching that. I forgot